tsconfig.json

```diff
@@ -34,5 +34,5 @@
 
     // "allowSyntheticDefaultImports": true,        /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */
   },
-  "include": ["./lib/**/*.js", "./test/**/*.spec.js", "./test/**/*.ts", "./benchmark/*.js"]
+  "include": ["./lib/**/*.js", "./test/**/*.spec.js", "./benchmark/*.js"]
```

Comment:
This is because we does not need to convert test files to JS to run them?

Reply:
Correct.  Jest handles this.  Keeping the CLI tests as .ts means they won't be rolled up into the web tests.

Reply:
Understood